main.go

```diff
@@ -449,12 +449,12 @@ func mainCore() error {
 
 	// Initial data summary for web ui. stakedb must be at the same height, so
 	// we get do this before starting the monitors.
-	blockData, _, err := collector.Collect()
+	blockData, val, err := collector.Collect()
```

Comment:
`val` -> `msgBlock` (or whatever doesn't clash with any other variables in `main`)

Reply:
ok. Noted